### PR TITLE
Add ildasm roundtrip

### PR DESCRIPTION
**Pull request by mmitche**
Also reworks some of the pri1 stuff to be cleaner

### CODE REVIEW

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -61,33 +70,113 @@ def static getBuildJobName(def configuration, def architecture, def os) {
...
+        switch (scenario) {
+            case 'default':
+                // default gets a push trigger for everything
+                Utilities.addGithubPushTrigger(job)
+                break
+            case 'pri1':
+                // Pri one gets a daily build, and only for release
+                if (architecture == 'x64' && configuration == 'Release') {
+                    // We don't expect to see a job generated except in these scenarios
+                    assert os == 'Windows_NT' || isFlowJob && os in crossList
```
Comment (kyulee1) [inline netci.groovy]: Should it be something like  os == 'Windows_NT' || (isFlowJob && os in crossList) ?

File: netci.groovy [hunk 2]
```diff
@@ -205,14 +307,21 @@ def static addPRTrigger(def job, def architecture, def os, def configuration, is
...
+                                            break
+                                        case 'pri1':
+                                            buildCommands += "build.cmd ${lowerConfiguration} ${architecture} Priority 1"
+                                            break
+                                        case 'ilrt':
+                                            // First do the build with skiptestbuild and then build the tests with ilasm roundtrip
+                                            buildCommands += "build.cmd ${lowerConfiguration} ${architecture} skiptestbuild"
+                                            buildCommands += "test\\buildtest.cmd ${lowerConfiguration} ${architecture} ilasmroundtrip"
+                                            break
+                                        default
```
Comment (kyulee1) [inline netci.groovy]: missing : for default

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Pull request merged**